File: 2-Ad-Hoc/ex2868.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    //variaveis
    int n, v1, v2, respCerto, respErrada, diferenca;
    char operacao, igual;
    char *errou;
    int i, j;

    //entrada
    scanf("%d", &n);

    for(j = 0; j < n; j++){
        scanf("%d %c %d %c %d", &v1, &operacao, &v2, &igual, &respErrada);

        //processamento
        if(operacao == '+'){
            respCerto = v1 + v2;
        }
        else if(operacao == '-'){
            respCerto = v1 - v2;
        }
        else{
            respCerto = v1 * v2;
        }

        diferenca = abs(respCerto - respErrada);
        
        errou = (char*) malloc(diferenca + 5);
        errou[0] = 'E';
        for(i = 1; i < diferenca + 1; i++){
            errou[i] = 'r';
        }
        errou[i] = 'o';
        errou[i + 1] = 'u';
        errou[i + 2] = '!';
        errou[i + 3] = '\0';

        //saida
        printf("%s\n", errou);
        free(errou);
    }
    return 0;
}

File: 2-Ad-Hoc/ex2868.py
#entrada
n = int(input())
for i in range(0, n):
    expressao = str(input()).split()

    #processamento
    v1 = int(expressao[0])
    operacao = expressao[1]
    v2 = int(expressao[2])
    respErrada = int(expressao[4])

    if operacao == '+':
        respCerta = v1 + v2
    elif operacao == '-':
        respCerta = v1 - v2
    else:
        respCerta = v1 * v2

    diferenca = abs(respCerta - respErrada)

    errou = 'E'
    errou += 'r' * diferenca
    errou += 'ou!'

    #saida
    print(errou)
